docs/fundamentals/code-analysis/quality-rules/ca1047.md

```diff
@@ -11,7 +11,6 @@ helpviewer_keywords:
 author: gewarren
 ms.author: gewarren
 dev_langs:
-- CSharp
```

Comment:
This is used because there's a C# code example. So I think we should keep this.

Reply:
```suggestion
- CSharp
```

Reply:
Re-added